main.js

```diff
@@ -0,0 +1,127 @@
+let key = openWeather.key;
```

Comment:
unless a value is reassigned using the assignment operator, they should be const instead of let